runtime/common/src/evm/precompile.rs

```diff
@@ -94,11 +99,75 @@ where
 	}
 }
 
+<<<<<<< HEAD
+/// A set of precompiles. This set might contain
+/// not yet mainnet ready precompiles in order to test
+/// those in development or staging environment without touching
+/// the mainnet set.
+pub struct Development<R>(PhantomData<R>);
```

Comment:
I'd probably implement this struct in terms of `CentrifugePrecompiles` so we can DRY up the list of common precompiles:

`pub struct Development<R>(CentrifugePrecompiles<R>);`
...